app/scripts/platforms/extension.js

```diff
@@ -160,10 +162,13 @@ export default class ExtensionPlatform {
 
     const url = getBlockExplorerLink(txMeta, rpcPrefs);
     const nonce = parseInt(txMeta.txParams.nonce, 16);
+    const view = startCase(
+      toLower(getURLHostName(url).replace(/([.]\w+)$/u, '')),
```

Comment:
I have to agree with @jpuri on this one. Your screenshot for Mumbai PolygonScan makes it appear that this works with subdomains (mumbai) but then Ehterscan would have a 'www' proceeding it? And if I do a naive assertion on this regex statement in console: 

```
const y = new URL('https://mumbai.polygonscan.com/');
> undefined
y.hostname.replace(/([.]\w+)$/u, '')
> 'mumbai.polygonscan'
y.hostname.replace(/([.]\w+)$/ug, '')
> 'mumbai.polygonscan'
```

Reply:
I missed the call to 'startCase' which is the reason the magic works. Alright i'm good with this. Thanks for pushing back @adnansahovic